rpm/qtsensors.spec

```diff
@@ -12,7 +12,8 @@ BuildRequires:  qt5-qtnetwork-devel
 BuildRequires:  qt5-qtdeclarative-qtquick-devel
 BuildRequires:  qt5-qmake
 BuildRequires:  fdupes
-BuildRequires:  pkgconfig(sensord-qt5)
+BuildRequires:  pkgconfig(sensord-qt5) >= 0.12.7
+#                                         ^^^^^^ = TBD
```

Comment:
These TBD versions are known after sensorfwd changes in: https://github.com/sailfishos/sensorfw/pull/9

Reply:
The required sensorfwd version is 0.13.0